web/annotation_editor_layer_builder.css

```diff
@@ -529,13 +529,52 @@
     mask-image: var(--alt-text-done-image);
   }
 
-  & .description {
-    position: absolute;
-    top: 0;
-    left: 0;
+  & .tooltip {
     display: none;
-    width: 0;
-    height: 0;
+
+    &.decorative {
+      font-style: italic;
+    }
+
+    &.show {
+      --alt-text-tooltip-bg: #f0f0f4;
+      --alt-text-tooltip-fg: #15141a;
+      --alt-text-tooltip-shadow: 0px 2px 6px 0px rgba(58, 57, 68, 0.2);
+
+      @media (prefers-color-scheme: dark) {
+        --alt-text-tooltip-bg: #1c1b22;
+        --alt-text-tooltip-fg: #fbfbfe;
+        --alt-text-tooltip-shadow: 0px 2px 6px 0px #15141a;
+      }
+
+      @media screen and (forced-colors: active) {
+        --alt-text-tooltip-bg: Canvas;
+        --alt-text-tooltip-fg: CanvasText;
+        --alt-text-tooltip-border: CanvasText;
+        --alt-text-tooltip-shadow: none;
+      }
+
+      display: inline-flex;
+      flex-direction: column;
+      align-items: center;
+      justify-content: center;
+      position: absolute;
+      top: calc(100% + 2px);
+      inset-inline-start: 0;
+      padding-block: 2px 3px;
+      padding-inline: 3px;
+      max-width: 300px;
+      width: max-content;
+      height: auto;
+      font-size: 12px;
+
+      border: 0.5px solid var(--alt-text-tooltip-border);
```

Comment:
It seems that this variable is now only set in forced-colors mode, and that there's no default value any more; I'm guessing that happened accidentally when updating the patch?